.travis.yml

```diff
@@ -1,27 +0,0 @@
-language: python
-python:
-- '2.7'
-- '3.6'
-- '3.7'
-- '3.8'
-dist: xenial
-before_install:
-    - pip install tox --upgrade
-    - pip install codecov --upgrade
-script: tox -e "$(echo py$TRAVIS_PYTHON_VERSION | tr -d .)-unit-functional"
-after_success:
-    - codecov -F unit -f cov_reports/unit.coverage.xml
-    - codecov -F functional -f cov_reports/functional.coverage.xml
-cache:
-  pip: true
-  directories:
-  - $TRAVIS_BUILD_DIR/.tox
-git:
-  depth: 2147483647
-deploy:
```

Comment:
Could you just keep the Travis tests for now ? I'll disable the requirement for these tests to pass.
But the deployment should be disabled.

Reply:
Oh, ok. My bad.